app.js

```diff
@@ -6,11 +6,25 @@ var http = require('http').Server(app);
 var io = require('socket.io')(http);
 
 var questionsList = question.correctQuestions();
+var question;
 
 var users = [];
 var leaders = [{name: 'Anton', points: 50}, {name: 'Pasha', points: 60}, {name: 'Maryna', points: 70}];
 
+function getQuestion(){
+    var generatedQuestion = questionsList[Math.floor(Math.random()*questionsList.length)];
+    question = generatedQuestion;
+    return generatedQuestion;
+}
 
+function checkTheAnswer(answer){
```

Comment:
@podviaznikov 
I know there is solution. But i forgot how it calls. It has such format a ? b : c
where a - condition, if it true func return b, if false - c.

Did you ask for that thing?